WordPressCom-Stats-iOS/Services/WPStatsServiceRemote.m

```diff
@@ -1067,16 +1067,12 @@ - (NSURLSessionDataTask *)operationForCountryForDate:(NSDate *)date
                 countryName = [countryInfoDict[key] stringForKey:@"country_full"];
             }
             StatsItem *statsItem = [StatsItem new];
-            statsItem.label = countryName;
+            statsItem.label = [NSString stringWithFormat:@"%@ %@",
```

Comment:
I think the RTL problem can be fixed by using `[NSString localizedStringWithFormat:]`